data/kernelci_api.py

```diff
@@ -0,0 +1,15 @@
+import urllib
```

Comment:
There is a `kernelci/data/kernelci_api.py` file in the `kernelci-core` repository.  So either we add support for getting a single object there, or we keep `get_node_by_commit_id()` in `trigger.py`.